Documentation/openshiftdocs/modules/remotewritespec.adoc

```diff
@@ -35,7 +35,7 @@ link:prometheusrestrictedconfig.adoc[PrometheusRestrictedConfig]
 
 |oauth2|*monv1.OAuth2|Defines OAuth2 authentication settings for the remote write endpoint.
 
-|proxyUrl|string|Defines an optional proxy URL.
+|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy if enabled.
```

Comment:
cc @eromanova97, whether the wording is ok for you.

Reply:
```suggestion
|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy, if enabled.
```
Just a very small suggestion :)